.github/actions/docker-build-image/action.yml

```diff
@@ -75,6 +75,7 @@ runs:
       run: |
         REPOSITORY=${DIR##*/}
         TAG="${TAG:-$(echo ${GITHUB_HEAD_REF:-$GITHUB_REF_NAME} | sed -e 's/[^-_.[:alnum:]]/_/g')}"
+        TAG="${TAG:0:128}"
```

Comment:
Nitpick: if the tag originally comes from the action input, I don't think we should modify it, and let it fail if it's too long.
For the fallback tag (second part of expression 1 line above), we can cut it, but I wonder if we need to use the branch name on PRs, maybe relying on GITHUB_REF_NAME for all use cases is better ("<pr_number>/merge" would be the value on PRs)

Reply:
thanks, set limitation only on generated value